src/view/configuration/configuration.jsx

```diff
@@ -153,6 +155,9 @@ const validateDuplicateValue = (createError, instances, key, message) => {
 const idSyncContainerIdValidationMessage =
   "Please specify a non-negative integer or data element for the container ID.";
 
+const onBeforeEventSendValidationMessage =
+  "Please specify a data element for the onBeforeEventSend callback.";
```

Comment:
Did we agree to wrap the vars in ` ` ? Like `onBeforeEventSend? cc: @Aaronius

Reply:
I'm glad you bring this up, because in this case, `onBeforeEventSend` means nothing to the user, since we don't use that label for the form field. The label @jileeshs chose to use for the form field is "Callback function for modifying data before each event is sent to the server". So either we say "Please specify a data element for the callback function for modifying data before each event is sent to the server" or we change the field's label to something simpler and use that in the validation message instead.

The validation errors point directly at the field with the problem, so having the name of the field doesn't provide much value and is one more place we need to keep updated when we modify field labels. At this point, I think it would be good to just drop the field labels from all the validation messages entirely. For example change, "Please specify a non-negative integer or data element for the container ID." to "Please specify a non-negative integer or data element." and change "Please specify a data element for the onBeforeEventSend callback." to "Please specify a data element." Let's do it across the whole extension though.

Unless someone disagrees?